components/net/tests/cookie.rs

```diff
@@ -398,7 +398,7 @@ fn test_cookie_eviction_all_secure_one_nonsecure() {
     let mut vec = Vec::new();
     for i in 1..5 {
         let st = format!(
-            "extra{}=bar; Secure; expires=Sun, 18-Apr-2026 21:06:29 GMT",
+            "extra{}=bar; Secure; expires=Sun, 18-Apr-2026 21:06:29:01 GMT",
```

Comment:
@mrobinson  I needed yo add this to make unit test pass

Reply:
Hrm...

Reply:
Yes, it's weird, but if I don't add seconds, none of these cookies will appear in the test result, and only one will be added: `foo=bar`

Reply:
This is very odd though because this isn't seconds but something else:

21:06:29:01 is 21 hours, 6 minutes, 29 seconds and 1...millisecond? I'm not sure what is going on here.

Reply:
Sorry I meant `millisecond`, yes apparently I need to add `millisecond`, I will debug it further

Reply:
found the issue `is_cookie_expired` is returning `true` but should be `false`